pkg/asset/templates/content/bootkube/etcd-ca-bundle-configmap.go

```diff
@@ -9,29 +9,29 @@ import (
 )
 
 const (
-	hostEtcdServiceOpenshiftEtcdFileName = "host-etcd-service.yaml"
+	etcdCAConfigMapFileName = "openshift-config-configmap-etcd-ca-bundle.yaml.template"
```

Comment:
nit: `etcd-ca-bundle.yaml.template` seems sufficient here, but I don't think that blocks this PR.